### PR TITLE
refactor: update ascat, delly, manta, TNscope_umi and tnhaplotyper tags 

### PR DESCRIPTION
**Pull request by ivadym**
### This PR:

- Removes the additional `sv` tag for `delly` and `manta` filtered files when running targeted analysis.
- Adds specific tags to the filtered `delly` and `ascat` files to distinguish them at CG level.
- Adds specific tags to the filtered `TNscope_umi` and `tnhaplotyper` files.
- Change tag name `cnvkit`.
- Adds specific tags to the filtered `tnscope` and `vardict` files.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by ivadym** [comment]
Tested HK file generation (`balsamic report deliver ...`) for `TN-PANEL` and `TN-WGS`: 

- Removes the additional `sv` tag for `delly` and `manta` filtered files when running targeted analysis:

`manta`
Screen Shot 2022-01-24 at 09 56 23

`delly`
Screen Shot 2022-01-24 at 09 57 56

- Adds specific tags to the filtered `delly` and `ascat` files to distinguish them at CG level.

`ascat`
Screen Shot 2022-01-24 at 10 51 19

`delly`
Screen Shot 2022-01-24 at 10 48 38

- Adds specific tags to the filtered `TNscope_umi` and `tnhaplotyper` files.

`tnhaplotyper`
Screen Shot 2022-01-24 at 10 39 44

`TNscope_umi`
Screen Shot 2022-01-24 at 10 38 57

File: BALSAMIC/snakemake_rules/annotation/varcaller_sv_wgs_filter_tumor_normal.rule [hunk 1]
```diff
@@ -48,16 +48,16 @@ rule bcftools_filter_delly:
...
+tabix -p vcf -f {output.vcf_sv_pass_delly};
     """
 
 
 rule bcftools_filter_ascat:
   input:
     vcf = vep_dir + "CNV.somatic.{case_name}.ascat.all.vcf.gz",
   output:
-    vcf_sv_pass = vep_dir + "CNV.somatic.{case_name}.ascat.all.filtered.pass.vcf.gz",
+    vcf_sv_pass_ascat = vep_dir + "CNV.somatic.{case_name}.ascat.all.filtered.pass.vcf.gz",
```
Comment (ashwini06) [inline BALSAMIC/snakemake_rules/annotation/varcaller_sv_wgs_filter_tumor_normal.rule:60]:
```suggestion
    vcf_cnv_pass_ascat = vep_dir + "CNV.somatic.{case_name}.ascat.all.filtered.pass.vcf.gz",
```
Can we be explicit on the var_type for ascat here?

**Review by ashwini06 — COMMENTED** [review]
Tag specification is much better :) just a minor comment for ascat cnv?

File: BALSAMIC/snakemake_rules/annotation/varcaller_sv_wgs_filter_tumor_normal.rule (hunk 1 above)
Comment (ivadym) [reply BALSAMIC/snakemake_rules/annotation/varcaller_sv_wgs_filter_tumor_normal.rule]: 👍 

I also updated accordingly the `cnvkit` var name (`varcaller_sv_filter.rule`).

**Review by ashwini06 — APPROVED** [review]
👍🏼

**Comment by ivadym** [comment]
Closing as @khurrammaqbool is merging the var callers outputs. However, the issue will remain open until the tags has been updated in hermes and cg.

**Pull request closed without merging**